server/chat-plugins/room-events.ts

```diff
@@ -52,7 +52,7 @@ export const commands: ChatCommands = {
 			} else if (room.eventsRank === '@') {
 				if (!this.can('ban', null, room)) return false;
 			} else if (room.eventsRank === '#') {
-				if (!this.can('declare')) return false;
+				if (!this.can('gamemanagement')) return false;
```

Comment:
again, why is this global?

Reply:
also did you mean to change it in the other places?

Reply:
yes, I did.  Dumb mistake.